ecosystem/sep-0009.md

```diff
@@ -81,6 +81,7 @@ Name | Type | Description
 `notary_approval_of_photo_id` | binary | Image of notary's approval of photo ID or passport
 `ip_address` | string | IP address of customer's computer
 `photo_proof_residence` | binary | Image of a utility bill, bank statement or similar with the user's name and address
+`sex` | string | `male`, `female`, or `other`
```

Comment:
I think the values included here make sense for now, given the field is `sex` and not `gender`. If the field was `gender` as @rice2000 suggested we would probably include some other values. There's some previous discussion on this at https://github.com/stellar/stellar-protocol/issues/915#issuecomment-827055684.

The primary use case of this field is to capture data that is verifiable on official documentation such as passports or other forms of photo identification. According to wikipedia there are countries that support three markers on passports: other (X), female (F), male (M), so the values in this PR appear to align well with what we can expect on passports.

We should add other values if we discover there are forms of identification that support additional values to keep this inclusive of all countries and their KYC requirements.